src/analyzing/Constant_propagation.ml

```diff
@@ -134,12 +134,32 @@ let is_js env =
   | Some lang -> Lang.is_js lang
 
 let is_global = function
-  | Global -> true
-  | _ -> false
```

Comment:
weird that the _else_ semgrep rule didn't find this code.

Reply:
Yes indeed. Maybe a bug in OCaml's support? 🤔